src/utils/ContractUtils.ts

```diff
@@ -82,3 +91,48 @@ export async function calculateEstimatedGas(
     .estimateGas({ from }, (err, estGas) => (err ? GASLIMIT_DEFAULT : estGas))
   return estimatedGas
 }
+
+/**
+ * Send the transation on chain
+ * @param {string} from account that calls the function
+ * @param {any} estGas estimated gas for the transaction
+ * @param {Web3} web3 web3 objcet
+ * @param {Function} functionToEstimateGas function that we need to send
+ * @param {...any[]} args arguments of the function
+ * @return {Promise<any>} transaction receipt
+ */
+export async function sendTx(
+  from: string,
+  estGas: any,
```

Comment:
as far as I see, this is always a number in our code so we should be able to be more precise than `any`

Reply:
true, will update this and also will try to make the methods wrapper methods for estimate and send more restrictive on the return type